### ISSUE TITLE
Bug: Hawaii and McKinney landing pages’ background are showing the wrong maps

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#1265]
#### Describe the bug 

hi.foodoasis.net should show a map of the Hawaiian islands in the background.
mckinney.foodoasis.net should show map of McKinney in the background

#### Steps to reproduce the issue

0. As seen on https://github.com/hackforla/food-oasis/wiki
1. Go to the following websites
[la.foodoasis.net](https://la.foodoasis.net/) or [foodoasis.la](https://foodoasis.la/) - Los Angeles County
[ca.foodoasis.net](https://ca.foodoasis.net/) - Areas of California outside of LA County
[hi.foodoasis.net](https://hi.foodoasis.net/) - The Hawaiian Islands
[sb.foodoasis.net](https://sb.foodoasis.net/) - Santa Barbara County
[mckinney.foodoasis.net](https://mckinney.foodoasis.net/) - McKinney, TX

3. Click on each of the links

#### What's the expected result?

- Each location should have its own map

At one time we had Hawaii and McKinney-specific background maps.  

Each map was hand-generated — they are static images used as backgrounds, not live maps.

I don’t know about Santa Barbara (sb.foodoasis.net) or beyond CA (ca.foodoasis.net) — they may not have been generated yet.

If maps don’t exist, perhaps use a plain white background as a placeholder, but it should not the Los Angeles map.

#### What's the actual result?

- All landing pages show the Los Angeles map, and not their specific maps.

#### Device configuration
- Mac Safari, iPad Safari

**Comment by fancyham** [comment]
These work properly (checked the dev site only — public sites are still currently wrong) — thanks for fixing them.
https://devhi.foodoasis.net
https://devmckinney.foodoasis.net

This one don't have the right map — I don't think we ever created them, though.  
So better question is should these be on the wiki page (https://github.com/hackforla/food-oasis/wiki)?
https://devca.foodoasis.net
https://devsb.foodoasis.net

**Comment by staceyrebekahscott** [comment]
@entrotech Reopened issue, as I just checked the Hawai'i public site: https://hi.foodoasis.net/, and the TX public site: https://mckinney.foodoasis.net/. The CA map still shows as the background image.

**Comment by entrotech** [comment]
The changes have not yet been released to production., so you will only see them in the dev environment until the next release.
As @fancyham mentioned, custom maps were only created for Los Angeles, Hawaii, and McKinney, so the remaining sites (Santa Barbara, California and Oregon) still show the Los Angeles map.

We do need to decide which sites to keep:

- We were working with the Code for Portland brigade of CfA for a while, but they determined that the Oregon Food Bank has a comprehensive pantry/ meal program listing [here](https://www.oregonfoodbank.org/find-support), and decided that a Food Oasis site for Oregon would be redundant. I think this site can be dropped.
- The California site was proposed by Joel Parker-Henderson (a former member of HfLA) during 2019, specifically to address food insecurity in Northern California due to the wildfires that year, but the scope was pretty large, and we were never able to collect comprehensive data across the state. We also ran into some resistance from food banks (i.e. the Redwood Food Bank) that did not want us publishing the data for their outlets, out of concern that we would not be able to keep our listings current on an ongoing basis. I think this site can be dropped.
- The Santa Barbara site was discussed between Jenny Mikesell, myself, Julia Blanton of Community Environmental Council 9https://cecsb.org/) , Hillary Si and Katie Maynard of UCSB. I created the site, but none of us really followed up to populate their site with data or market it.  I think there is an opportunity there, as these folks seem to be pretty plugged into the food insecurity and related organizations in Santa Barbara county. I think we should get back in touch with these folks, and see if we can get it going.
